docker/.env

```
@@ -0,0 +1,14 @@
+# ros vars
+ROS_DOMAIN_ID=31
+POSE_EST_SRC_DIR=/home/coresense/workspaces/erf_ws/src/ros2-aruco-pose-estimation
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSE_EST_SRC_DIR key should go before the ROS_DOMAIN_ID key

(UnorderedKey)